crypto.go

```diff
@@ -72,7 +74,15 @@ func (i *Identity) ConfigForPeer(remote peer.ID) (*tls.Config, <-chan ic.PubKey)
 	conf := i.config.Clone()
 	// We're using InsecureSkipVerify, so the verifiedChains parameter will always be empty.
 	// We need to parse the certificates ourselves from the raw certs.
-	conf.VerifyPeerCertificate = func(rawCerts [][]byte, _ [][]*x509.Certificate) error {
+	conf.VerifyPeerCertificate = func(rawCerts [][]byte, _ [][]*x509.Certificate) (err error) {
```

Comment:
This is not needed, since we have the panic handler in `handshake`, is it?

Reply:
- This way we don't panic _through_ TLS (not sure what that might do).
- E.g., I don't trust that TLS won't run this on another goroutine.

Reply:
crypto/tls doesn't, but we can leave it here to be extra paranoid.